epics-team-builder/src/widgets/SetupPage/components/SetupPage.jsx

```diff
@@ -11,15 +12,17 @@ class SetupPage extends React.Component {
   };
 
   render() {
-    const { numOfPreferredProjects } = this.props;
+    const { numOfPreferredProjects, students, projects, assignProjToStud } = this.props;
 
     return (
       <div className='setup-page'>
         <Header />
         <div className='setup-grid'>
           {/* Make sure to put these divs in their respective components when made */}
           <ExcelReader />
-          <div className='manual-project-assignment'></div>
+          <div className='manual-project-assignment'>
```

Comment:
I would just move the className to inside the ManuallyAssignProjects component. Theres already a div wrapping it internally which you can move it too.